redisvl/extensions/router/schema.py

```diff
@@ -63,27 +63,28 @@ class DistanceAggregationMethod(Enum):
 class RoutingConfig(BaseModel):
     """Configuration for routing behavior."""
 
-    distance_threshold: float = Field(default=0.5)
+    # distance_threshold: float = Field(default=0.5)
     """The threshold for semantic distance."""
     max_k: int = Field(default=1)
-    """The maximum number of top matches to return."""
+
+    """Aggregation method to use to classify queries."""
     aggregation_method: DistanceAggregationMethod = Field(
         default=DistanceAggregationMethod.avg
     )
-    """Aggregation method to use to classify queries."""
+
+    """The maximum number of top matches to return."""
+    distance_threshold: float = Field(
+        default=0.5,
+        deprecated=True,
```

Comment:
On a v1 BaseModel, this only adds "deprecated" to the JSON schema for this field, since what's really happening is any extra keyword args pass through to the schema. That's not ideal, but I'm ok with it. In v2, this would log a deprecation warning if a user used the field. We can leave it in place like this and then if it's still around when we cut over to using v2 BaseModel, it'll work better.